poppy/zernike.py

```diff
@@ -1098,8 +1105,8 @@ def opd_expand_nonorthonormal(opd, aperture=None, nterms=15, basis=zernike_basis
     return coeffs
 
 
-def opd_from_zernikes(coeffs, basis=zernike_basis_faster, aperture=None, outside=np.nan,
-                      **kwargs):
+def compose_opd_from_basis(coeffs, basis=zernike_basis_faster, aperture=None, outside=np.nan,
+                           **kwargs):
```

Comment:
The example in the docstring for `compose_opd_from_basis` still calls the old function name